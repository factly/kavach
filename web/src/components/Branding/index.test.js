import React from 'react';
import { shallow } from 'enzyme';

import BrandingComponent from "./index";

describe("BrandingComponent", () => {
	it("should render the component", () => {
		let component = shallow(<BrandingComponent />);
		expect(component).toMatchSnapshot();
	});
});
