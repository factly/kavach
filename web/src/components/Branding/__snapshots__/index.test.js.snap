// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrandingComponent should render the component 1`] = `
<div
  style={
    Object {
      "background": "#3A54AA",
      "display": "flex",
      "justifyContent": "center",
      "width": "50%",
    }
  }
/>
`;
